### PR TITLE
TC_02.04.018 | Dashboard > Verify Command panel on the left side

### PR DESCRIPTION
**Pull request by JackBlaaack**
https://trello.com/c/jhnEaNb3/208-tc0204018-dashboard-verify-command-panel-on-the-left-side

### CODE REVIEW

**Comment by gurinowicz** [comment]
[TC_03.08.008 | +New Item > Copy from > User is able to create a New job](https://trello.com/c/k4LAg4eV/157-tc0308008-new-item-copy-from-user-is-able-to-create-a-new-job)

**Review by ElenaDetusheva — APPROVED** [review]

**Comment by 7108581** [comment]
We store test data in fixtures. 
Please, place the 'expectedText' in the fixtures.

File: cypress/e2e/checkLeftSideBar.cy.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress"/>
+
+describe('Check left side bar', () =>{
+    it('TC_02.04.018 | Dashboard > Verify Command panel on the left side', () =>{
+    cy.get('#tasks').each(($element) =>{
```
Comment (Rom-T) [inline cypress/e2e/checkLeftSideBar.cy.js:5]: There is no need to use the each() method, there is only 1 element. You're better off using then()

Comment (JackBlaaack) [reply cypress/e2e/checkLeftSideBar.cy.js:5]: Explain, please, I think I check all of elements in command pannel

Comment (JackBlaaack) [reply cypress/e2e/checkLeftSideBar.cy.js]: I see

**Review by Paul-Korz — APPROVED** [review]

**Review by RuslanG13 — APPROVED** [review]

**Pull request merged**